tests/utilities/nidigital_measurement/__init__.py

```diff
@@ -38,12 +39,20 @@ def measure(
                 connections = reservation.get_nidigital_connections(pin_names)
                 assert all([session_info is not None for session_info in session_infos])
                 passing_sites, failing_sites = _burst_spi_pattern(session_infos)
+                site0_connections, site1_connections = [], []
```

Comment:
FYI, these are channel names, not connections, but my other suggestion removes these variables.

Reply:
Yes, removed them.